timesketch/lib/analyzers/sequence_sessionizer_test.py

```diff
@@ -170,89 +170,104 @@ def test_session_type_empty_str(self):
             sessionizer.run()
 
 
-class BaseManyEventsSequenceSessionizerPlugin(object):
-    """Tests base functionality of sequence sessionizing sketch analyzer with
-    many events in the even_seq."""
+class TestManyEventsSequenceSessionizerPlugin(BaseTest):
+    """Tests base functionality of sequence sessionizing sketch analyzers with
+    many events in the even_seq which are listed in seq_sessionizer_classes.
+
+        Attributes:
+            seq_sessionizer_classes: A list of sequence sessionizer classes to
+                test.
+    """
+    seq_sessionizer_classes = [ManyEventsSequenceSessionizer]
```

Comment:
Same comment as above, please extend the docstring to mention something about adding new classes here